Sage/carbon
# fix(date): ensure that contrast ratio for weekday meets AA standards

Changing the token to actionMinor500 means we will meet AAA standards.

fixes #5173

### Proposed behaviour

![date-input-actionMinor500](https://user-images.githubusercontent.com/56251247/173386103-5d1151ac-622f-424b-b7ff-73bea9ab99e1.png)

### Current behaviour

![date-input-actionMinor400](https://user-images.githubusercontent.com/56251247/173386147-34b9763d-304a-44a8-a8f8-e156355fea2f.png)

### Checklist




#### QA


### Additional context



### Testing instructions



- Checkout the branch
- Start Storybook
- Navigate to Date component
- Open calendar and inspect the weekday text
- should be actionminor500 token
